extensions/amp-analytics/cross-origin-tracking.md

```diff
@@ -0,0 +1,18 @@
+# Tracking users across origins
+
+AMP pages can be served from either a publisher's website or the AMP Cache.  Because these two websites have different domains, their access to each other’s cookies can be limited in the context of some browsers.  These additional security restrictions mean that some additional  effort will be required to maintain the client identification if and when users browse from both contexts.
+ 
+AMP-Analytics makes this effort simpler by providing a [client identifier as a variable substitution](analytics-vars.md#clientid).  The clientId variable can be called with the cid-scope parameter, which is the name of a cookie to read from if the document is loaded from a first-party context.  Otherwise, the clientId variable will be replaced with a cookie set by AMP itself.  At this moment, AMP doesn't provide for direct access to client-side storage (i.e. cookies).
+
+However, by sending the collection request to your own domain you can read and write cookies on those requests which enables access to both the AMP clientId and your cookies in the same request.
```

Comment:
maybe add something like: "Once the analytics request reaches your server, it can modify the request as needed and respond with [302 response](https://en.wikipedia.org/wiki/HTTP_302) code and [set-cookie headers](https://en.wikipedia.org/wiki/HTTP_cookie#Setting_a_cookie) and redirect the users to the correct destination." 

@rudygalfi does that make sense?